java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/internal/Expectations.java

```diff
@@ -180,17 +189,77 @@ public String toString() {
     }
   }
 
+  private static class EditComment {
+    final Map<IssueAttribute, Object> attributes;
+
+    public EditComment(Issue issue) {
+      attributes = issue;
+    }
+
+    String replacement() {
+      String replacement = (String) attributes.get(MESSAGE);
```

Comment:
We can use directly the method from the enum to avoid the cast
`String replacement = MESSAGE.get(attributes);`